llvm/test/MC/Disassembler/X86/apx/wrssq.txt

```diff
@@ -0,0 +1,6 @@
+# RUN: llvm-mc --disassemble %s -triple=x86_64 | FileCheck %s --check-prefixes=ATT
+# RUN: llvm-mc --disassemble %s -triple=x86_64 -x86-asm-syntax=intel --output-asm-variant=1 | FileCheck %s --check-prefixes=INTEL
```

Comment:
Could you merge the test  WRUSSD with WRUSSQ? `D`, `Q` here is kind of a suffix here.

Reply:
They are listed as different instructions in document.